bin/peas_shell

```diff
@@ -269,6 +331,8 @@ class PanSensorShell(cmd.Cmd):
             if not delay:
                 delay = self._loop_delay
 
+            # WARNING: It appears we have a single _timer attribute, but we create
```

Comment:
Yikes. Good catch. Want to change this now?

Reply:
No, I want to figure out a better design first (e.g. a separate process for each board and another for the weather sensor).